content/sensu-go/5.12/installation/auth.md

```diff
@@ -1094,6 +1099,412 @@ example      | {{< highlight shell >}}"name": "activedirectory"{{< /highlight >}
 
 See the [LDAP troubleshooting](#ldap-troubleshooting) section.
 
+## OIDC authentication
+
+The Sensu offers license-activated support for OIDC driver for using the OpenID Connect 1.0 protocol (OIDC) on top of the OAuth 2.0 protocol for RBAC authentication.
+
+### OIDC configuration examples
+
+
+{{< language-toggle >}}
+
+{{< highlight yml >}}
+---
+type: oidc
+api_version: authentication/v2
+spec:
+  oidc:
+    additionalScopes:
+    - groups
+    clientId: a8e43af034e7f2608780
+    clientSecret: b63968394be6ed2edb61c93847ee792f31bf6216
+    insecure: false
+    redirectURL: http://127.0.0.1:4000/login/callback
+    server: https://localhost:9031
+    roles:
+    - name: guests
+      members:
+      - myorganization/guests
+      datacenters:
+      - us-west-1
+      subscriptions:
+      - webserver
+      readonly: true
+    - name: operators
+      members:
+      - myorganization/operators
+      datacenters: []
+      subscriptions: []
+      readonly: false
+  metadata:
+    name: oidc
+{{< /highlight >}}
+
+{{< highlight json >}}
+{
+  "type": "oidc",
+  "api_version": "authentication/v2",
+  "spec": {
+    "oidc": {
+      "additionalScopes": [ "groups" ],
+      "clientId": "a8e43af034e7f2608780",
+      "clientSecret": "b63968394be6ed2edb61c93847ee792f31bf6216",
+      "insecure": false,
+      "redirectURL": "http://127.0.0.1:4000/login/callback",
+      "server": "https://localhost:9031",
+      "roles": [
+        {
+          "name": "guests",
+          "members": [
+            "myorganization/guests"
+          ],
+          "datacenters": [
+            "us-west-1"
+          ],
+          "subscriptions": [
+            "webserver"
+          ],
+          "readonly": true
+        },
+        {
+          "name": "operators",
+          "members": [
+            "myorganization/operators"
+          ],
+          "datacenters": [],
+          "subscriptions": [],
+          "readonly": false
+        }
+      ]
+    },
+  	"metadata": {
+    	"name": "oidc"
+  	}
+	}
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+### RBAC for OIDC definition specification
+
+#### `oidc` attributes
+
+additionalScopes | 
+-----------------|------
+description      | Scopes to include in the claims, in addition to the default `email`, `openid` and `profile` scopes. _NOTE: only required for certain OIDC providers, such as Okta._
```

Comment:
There is now only the `openid` scope that's there by default (in line with the OIDC spec). If the user needs `email`, `profile`, ... or anything else they need to add it explicitly.